content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
```

Comment:
We talked about adding an additional sentence to augment the beta tag... something like "The Group Export operation is available for testing in the public sandbox, but is not generally available for use in client production environments. "

Reply:
I have added this note to the end of the Overview here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR20. Let me know if I should move it to the beginning.

Reply:
> I have added this note to the end of the Overview here: [eff91ed#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR20](https://github.com/cerner/fhir.cerner.com/commit/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR20). Let me know if I should move it to the beginning.

I think we may want to move it to the very first sentence under Overview - so that no one misses it and hopefully it will be easier for us to find when we need to remove it.

Reply:
Moved it to the beginning of the overview: https://github.com/cerner/fhir.cerner.com/pull/899/commits/780d6472fc27b87e37ef7361457f44e8da969b69#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR14